_templates/layout.html

```diff
@@ -67,6 +67,35 @@
 
   {% block body %}{% endblock %}
 </div>
+
+{% if (not meta or meta.get('allow_comments') != 'False') and display_comments %}
+<hr>
+<h2>User-contributed notes</h2>
+<p>
+  <em>Please read the <a href="https://github.com/Calinou/godot-docs/discussions/46">User-contributed notes policy</a> before submitting a comment.</em>
+</p>
+{# Use https://giscus.app/ to regenerate the script tag if needed. #}
+{# data-term is set to be language-independent and version-independent, so that comments can be centralized for each page. #}
+{# This increases the likelihood that users will encounter comments on less frequently visited pages. #}
+<script src="https://giscus.app/client.js"
+  data-repo="Calinou/godot-docs"
```

Comment:
```suggestion
  data-repo="godotengine/godot-docs"
```
Something worth investigating for future followups would be to see if we could automatically attach a small markup indicating the branch/version the comment was left with, i.e. adding "Godot 4.1" in a subtle font next to the comment's username for a comment left via a 4.1 branch page. (`stable` would of course complicate this)

Reply:
I need to regenerate the HTML tag using https://giscus.app (so that `data-repo-id` is also changed), but I need the Discussions feature on godotengine/godot-docs to be enabled first:

![image](https://github.com/godotengine/godot-docs/assets/180032/c0b24835-6730-47ed-b8aa-f2ad1cb7f0d5)

I don't have permission to do this.